### DISCUSSION TITLE
Releasing more often / smaller increments? 🔁

### DISCUSSION THREAD

**Discussion by nickluger** [directus/directus#5970] [Feature Requests]
I saw that the rc.70 milestone for example is currently accumulating a lot of largish tasks & PRs, while containing a lot of crucial bugfixes and issues, too. It was previously at 100%, before new (rather big) features creeped in. The last release was 14 days ago. Would it be possible to release more often or separate issue fixes from new features? Or do these things currently depend on each other?

Just a suggestion, born out of the fact, that I'm waiting for some crucial fixes and features and keep pinging the repo to see, whether the release is ready 😅

**Comment by rijkvanzanten** [comment]
We normally do that, but in this case, this is correct: "Or do these things currently depend on each other?"

Sometimes, a release includes fixes / features that needs more real life testing before we can safely distribute it to the world

**Reply by nickluger** [reply]
OK, thank you for the explanation @rijkvanzanten !
